website/docs/cloud-docs/workspaces/settings/vcs.mdx

```diff
@@ -4,82 +4,81 @@ page_title: VCS Connections - Workspaces - Terraform Cloud and Terraform Enterpr
+You can use trigger patterns and trigger prefixes in the **Add path** field to specify groups of files and directories.
+- **Trigger Patterns:** (Recommended) Use glob patterns to specify the files that should trigger a new run. For example, `/submodule/**/*.tf`, specifies all files with the `.tf` extension that are nested below the `submodule` directory. You can also use more complex patterns like `/**/networking/**/*`, which specifies all files in the `networking` folder at this level of the file path (e.g., `/submodule/service-1/networking/private/main.tf`). Refer to [Glob Patterns for Automatic Run Triggering](#glob-patterns-for-automatic-run-triggering) for details.
+- ** Trigger Prefixes:** Terraform Cloud will queue runs for changes in any of the specified trigger directories matching the provided prefixes (including the working directory). For example, if you use a top-level `modules` directory to share Terraform code across multiple configurations, changes to the shared modules are relevant to every workspace that uses that repo. You can add `modules` as a trigger directory for each workspace to make sure they notice any changes to shared code.
 
-## Automatic Speculative Plans
+-> **Note:** Terraform Cloud triggers runs on all attached workspaces if it does not receive a list of changed files or if that list is too large to process. When this happens, Terraform Cloud may show several runs with completed plans that do not result in infrastructure changes.
 
-Whether to perform [speculative plans on pull requests](/cloud-docs/run/ui#speculative-plans-on-pull-requests) to the connected repository, to assist in reviewing proposed changes.
+### VCS Branch
 
-Automatic speculative plans are enabled by default, but you can disable them for any workspace.
+This designates which branch of the repository Terraform Cloud should use in the workspace. If left blank, Terraform Cloud uses the repository's default branch.
 
-## VCS Branch
+### Automatic Speculative Plans
 
-Which branch of the repository to use. If left blank, Terraform Cloud will use the repository's default branch.
+Whether to perform [speculative plans on pull requests](/cloud-docs/run/ui#speculative-plans-on-pull-requests) to the connected repository, to assist in reviewing proposed changes. Automatic speculative plans are enabled by default, but you can disable them for any workspace.
 
-## Include submodules on clone
+### Include Submodules on Clone
 
-Whether to recursively clone all of the repository's Git submodules when fetching a configuration.
+Select **Include submodules on clone** to recursively clone all of the repository's Git submodules when Terraform Cloud feteches a configuration.
 
 -> **Note:** The [SSH key for cloning Git submodules](/cloud-docs/vcs/#ssh-keys) is set in the VCS provider settings for the organization, and is not related to the workspace's SSH key for Terraform modules.
+
+## Glob Patterns for Automatic Run Triggering
```

Comment:
@matejrisek added this here. Refer to my earlier comment but TLDR is that if this is really only relevant to trigger patterns and NOTHING else in TFC, maybe we should document it on this page?

Reply:
It is relevant only to trigger patterns atm. We could add it here for starters. Great idea 🥇